src/foam/dao/EasyDAO.js

```diff
@@ -632,19 +631,12 @@ model from which to test ServiceProvider ID (spid)`,
       class: 'Boolean',
       documentation: `
         Denotes if a model is approvable aware, and if so it should ALWAYS have this decorator on,
-        if an object is approvableAware but the user does not want the approval requests turned on,
-        they should set the approvableAwareEnabled property to false instead of setting the
-        approvableAware property to false
-      `,
-      javaFactory: 'return getEnableInterfaceDecorators() && foam.nanos.approval.ApprovableAware.class.isAssignableFrom(getOf().getObjClass());'
-    },
-    {
-      name: 'approvableAwareEnabled',
```

Comment:
Need to make sure with @jlhughes no service journals exist with .setApprovableAwareEnabled(false) on prod otherwise the servers won't build. Think we can do what I did for deletedAware where we leave it on the EasyDAO but just make it do nothing and be an empty boolean.

Reply:
@nanoscott

Reply:
We'd have to write another shell script to completely scrape out .setApprovableAwareEnabled(true) if they exist in prod journals in addition to all other services references to the method before we merge it in and take it to prod

Reply:
re: do nothing / empty implementation - please.

Reply:
Fixed in https://github.com/foam-framework/foam2/pull/3422/commits/bfe58b50665950d85bd6be00059112e2a90725ad.